packages/lit-element/.gitignore

```diff
@@ -0,0 +1,30 @@
+/node_modules/
```

Comment:
Remove this file. Make sure the root `.gitignore` is ignoring the right files. One reason I moved lit-html to a root `index.js` output is so that there's a consistent pattern to ignore.

Reply:
Done.